src/configobj/__init__.py

```diff
@@ -2001,8 +1993,8 @@ def write(self, outfile=None, section=None):
             self.indent_type = DEFAULT_INDENT_TYPE
 
         out = []
-        comment_markers = [self._a_to_u(x) for x in self.COMMENT_MARKERS]
-        comment_marker_default = self._a_to_u(self.COMMENT_MARKERS[0] + ' ')
+        comment_markers = [x for x in self.COMMENT_MARKERS]
```

Comment:
Since you wanted to remove redundancy, this is just `list(…)`.

Reply:
I've changed it to `tuple(self.COMMENT_MARKERS)` as that lets the code later use the tuple argument form of `str.startswith` that Python 2.5 introduced.